payroll/models/hr_payslip.py

```diff
@@ -169,47 +164,41 @@ class HrPayslip(models.Model):
         compute="_compute_payslip_count", string="Payslip Computation Details"
     )
     hide_child_lines = fields.Boolean(string="Hide Child Lines", default=False)
+    hide_invisible_lines = fields.Boolean(string="Hide Invisible Lines", default=False)
```

Comment:
please change the string to "Show only lines that appear on payslip" so it's more clear to the user.

Reply:
Done